requirements.txt

```
@@ -0,0 +1,7 @@
+gnomad_hail
```

```suggestion
git+https://github.com/macarthur-lab/gnomad_hail.git@master
```

Since gnomad_hail is not published yet, we'll need to use the git format for it.

https://pip.pypa.io/en/stable/reference/pip_install/#git